kong/templates/nginx_kong.lua

```diff
@@ -57,6 +56,14 @@ init_worker_by_lua_block {
     kong.init_worker()
 }
 
+upstream kong_upstream {
+    server 0.0.0.1;
+    balancer_by_lua_block {
+        kong.balancer()
+    }
+    keepalive 60;
```

Comment:
> With different keepalive settings.

I think that making it globally configurable in the configuration file would be enough for now.